### ISSUE TITLE
Unable to download the GeoLite DB

### ISSUE THREAD

**Issue by kevinbungeneers** [Intracto/SecretSanta#491]
Hi!

While taking care of #471, I noticed that the GeoLite DB isn't available for download anymore. After inspection I learned that this change was made deliberately by the people behind the database: 
> Starting December 30, 2019, we will be requiring users of our GeoLite2 databases to register for a MaxMind account and obtain a license key in order to download GeoLite2 databases. We will continue to offer the GeoLite2 databases without charge, and with the ability to redistribute with proper attribution and in compliance with privacy regulations. 

The whole article can be found at https://blog.maxmind.com/2019/12/18/significant-changes-to-accessing-and-using-geolite2-databases/

Would it be an acceptable fix to make the GeoIP database dependency an optional one? At least in the dev environment.

**Comment by tvlooy** [comment]
yes seems fine to me. Put install instructions in README. If the file is present we use it, if not we skip it

**Comment by tvlooy** [comment]
We work around the issue now, probably non compliant with the license but ot is what it is until we remove it as a dependency like stated aboven
